app/pages/login/page.tsx

```diff
@@ -53,6 +54,11 @@ export default function Login() {
                 <u>Sign Up</u>
               </button>
             </div>
+            <div className="flex-1 flex flex-row w-full justify-center items-center gap-2 text-foreground">
+              <Link href="/pages/sendResetEmail">
+                <u>Forgot Password</u>
```

Comment:
I realise the `Sign Up` option uses `<u>`, but it would be better to add the underline with CSS, right?

Reply:
Would that end up requiring more divs because of how the styling works for this setup?